ebusd-2.1.x/de/vaillant/08.hmu.csv

```diff
@@ -8,6 +8,12 @@ r,,EnergyCool,,,,,01,,,energy,,,,,,,,,,,,,,,
 r,,Hours,,,,,40,,,energy,,,,,,,,,,,,,,,
 r,,HoursHc,,,,,41,,,energy,,,,,,,,,,,,,,,
 r,,HoursCool,,,,,43,,,energy,,,,,,,,,,,,,,,
+
+# some more statistics (Live Monitor)
+r,,CurrentYieldPower,,,,,23,,,D1B,10,,,,,,,,,,,,,,,,,,,,
```

Comment:
* please add a comment for this (what does it reveal?)
* please add a unit (e.g. kW)
this applies to all added lines

Reply:
I made new commits and fixed all your requested changes. I have also added a new Value: "Water Throughput". As this is only my second time of using Git / GitHub, I am unsure what exactly I have to do to get this PR right. I have committed all my changes to this branch, can you see the changes?